project/src/const.ts

```diff
@@ -0,0 +1,14 @@
+export enum AppRoute {
+  Root = '/',
+  AddReview = '/add-review',
+  Film = '/films',
+  MyList = 'my-list',
+  Player = '/player',
+  SignIn = '/sign-in'
+}
```

Comment:
Рекомендую тут указать полные пути до страниц. С этим будет проще работать. Иначе тебе прийдётся дублировать логику про оставшуюся часть url'а во всех местах, где нужно будет получить путь до страницы.
Также заметил, что не все адреса указаны корректно.
```suggestion
export enum AppRoute {
  Root = '/',
  AddReview = '/films/:id/review',
  Film = '/films/:id',
  MyList = '/mylist',
  Player = '/player/:id',
  SignIn = '/login'
}
```